nucleus/pom.xml

```diff
@@ -190,7 +190,7 @@
         <gmbal.version>4.0.0-b001</gmbal.version>
         <antlr.version>2.7.7</antlr.version>
         <jersey.version>2.25.1</jersey.version>
-        <jackson.version>2.8.8</jackson.version>
+        <jackson.version>2.9.3</jackson.version>
```

Comment:
2.9.4 is the latest patch

Reply:
So it is!
https://github.com/FasterXML/jackson-databind/releases